src/main/webapp/availability.js

```diff
@@ -37,9 +37,21 @@ function loadAvailabilityTable(tableDiv, timezoneOffset) {
     .then(response => response.text())
     .then(tableContents => {
       tableDiv.innerHTML = tableContents;
+      makeEmptyRowsShort();
     });
 }
 
+function makeEmptyRowsShort() {
+  let rows = document.getElementsByTagName('tr');
+  for (let row of rows) {
+    if (row.firstElementChild.innerHTML === '') {
```

Comment:
Weird check, can we just apply the classes through JSP?

Reply:
Done through JSP now.